service/verifier/verify_test.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifier
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/hyperledger/fabric-x-committer/api/protoblocktx"
+	"github.com/hyperledger/fabric-x-committer/loadgen/workload"
+	"github.com/hyperledger/fabric-x-committer/utils/logging"
+)
+
+func TestVerifyBadTxForm(t *testing.T) {
```

Comment:
There is already a test that checks badly formed txs -- [TestBadTxFormat](https://github.com/hyperledger/fabric-x-committer/blob/fd133090bdb4365cd1de4ff97ae33cc04e128aec/service/verifier/verifier_server_test.go#L124)

Reply:
This test only verifies the verify method directly. Without streams and all that.

Reply:
Okay. Looks redundant. Anyway, when you move this to sidecar, the redundancy would be removed.

Reply:
Yes. It would be converted to a block mapping test, which is missing.